### DISCUSSION TITLE
Migrating Data from Chatwoot 2.10 to 3.0 on a New Server

### DISCUSSION THREAD

**Discussion by SyedAsadRazaDevops** [chatwoot/chatwoot#7849] [General]
I encountered an issue after updating Chatwoot from version **2.10** to **3.0** on my old server. The '**cwctl --upgrade**' script ran successfully, but during the migration command, it inadvertently deleted my data. 

As a result, I deployed a new server. Fortunately, I had backed up my PostgreSQL data from the old server, and it wasn't entirely erased. I confirmed that the conversation data still exists in the server's database. Now, I'm seeking guidance on how to migrate this 2.10 old server data to my newly deployed Chatwoot 3.0 server. 

**What's the recommended approach for this data migration?**

**Comment by sojan-official** [comment]
You will have to do the upgrade sequentially, going over 1-2 versions each at a time.
